math/R-app/files/R-app-helper.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [ -n "$DYLD_LIBRARY_PATH" ]; then
+   DYLD_LIBRARY_PATH=/opt/local/lib/libgcc:${DYLD_LIBRARY_PATH}
+else
+   DYLD_LIBRARY_PATH=/opt/local/lib/libgcc
```

Comment:
`/opt/local` is one of possible prefix. I suggest to use the real prefix instead ;)

Reply:
@catap Oops, of course. Error while copying. Will fix now.